src/main/java/pages/PaymentPage.java

```diff
@@ -0,0 +1,22 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class PaymentPage {
+    protected WebDriver driver;
+private By checkbox = By.id("billing-address-same-as-shipping-checkmo");
+private By totalAmount = By.className("amount");
+
+public PaymentPage (WebDriver driver){
+    this.driver=driver;
+}
+
+public void getCheckBox(){
+    driver.findElement(checkbox).getAttribute("checked").equals("true");
+}
```

Comment:
I would like to know what the purpose of the method is, because the name does not tell me what it does, apart I see that it is a "get" but it does not return anything

Reply:
Done

Reply:
You can't get the check box, I think you didn't read what I told you yesterday on slack, we agreed that `isShippingAddressActive()` is called and it returns a `boolean`, because this method doesn't return anything